src/components/ProjectCard.tsx

```diff
@@ -70,10 +72,29 @@ const renderButtons = (project: Project, loginLink: any) => {
   }
 }
 
+// Not sure what type the method handleContributorsClick should be classified as for TypeScript
+const renderContributorExpansionIcons = (contributorsLength: Number, handleContributorsClick: any) => {
+
+  if (contributorsLength > 6) {
+    return <div onClick={handleContributorsClick} className="card-contributor-icon-expand">
+      <FontAwesomeIcon icon={faArrowsAltV} />
```

Comment:
Instead of a single arrows icon, this should be two separate elements - [caret down](https://fontawesome.com/icons/caret-down?style=solid) when contributors are collapsed, and [caret up](https://fontawesome.com/icons/caret-up?style=solid) when contributors are expanded 😊

Reply:
Thank you @partiallymisplaced ! I've updated my code to use the two separate icons for expanding and collapsing. Here are some screenshots:
<img width="379" alt="Screen Shot 2020-11-01 at 1 44 25 PM" src="https://user-images.githubusercontent.com/34215359/97811663-06130700-1c4a-11eb-9315-a31048d849cc.png">
<img width="392" alt="Screen Shot 2020-11-01 at 1 44 19 PM" src="https://user-images.githubusercontent.com/34215359/97811664-07dcca80-1c4a-11eb-99b3-7c2c90c07e19.png">